package.json

```diff
@@ -1697,6 +1703,15 @@
                     "dark": "resources/dark/settings.svg"
                 }
             },
+            {
+                "command": "vscode-docker.networks.create",
+                "title": "Create...",
+                "category": "Docker Networks",
+                "icon": {
+                    "light": "resources/light/connect.svg",
```

Comment:
For icons I think we should use the network icon with a + superimposed, sort of like what is used for files:
![image](https://user-images.githubusercontent.com/36966225/67106220-2c2d4c80-f198-11e9-9ec2-042af330ba38.png)
I'm not entirely sure how we go about getting such a thing created though... @EricJizbaMSFT do you know?

Reply:
I'm happy to make one like that if there is no more official process.

Reply:
@misolori I saw you created most of the current icons, are you able to make one like this? The + sign superimposed over the network icon?

Reply:
Check here for icons, but not sure the one you're describing exists: https://github.com/microsoft/vscode-icons

I feel like you could just use the "plus" icon (which definitely exists in that repo) since "network" is implied and unlike the file view in the above screenshot, there's only one thing you can create

Reply:
Yup, that's reasonable as well.

Reply:
+1 on using the plus instead, otherwise it's a bit of a complicated icon:

<img width="229" alt="image" src="https://user-images.githubusercontent.com/35271042/67113689-fafc4e80-f18e-11e9-827c-290dc06f7383.png">